src/main/webapp/mapsScript.js

```diff
@@ -0,0 +1,79 @@
+ /*
+ * Copyright 2016 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the
+ * License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Fetch apiKey and dynamically load the Maps JavaScript API.
+ */
+function loadMapsScript() {
+  fetch('apiKey.json').then(response => response.json()).then(jsonResponse => {
+    GoogleMapsApiKey = jsonResponse.apiKey;
+    // Create the Google Maps tag with the fetched api key.
+    let script = document.createElement('script');
```

Comment:
Generally speaking we like to use `const` if we never need to reassign `script`.

it is a good habit to get into always using `const` by default, and only using `let` as an exception.

```suggestion
    const script = document.createElement('script');
```

Reply:
Thanks, I changed it to const.